server/models/Item.js

```diff
@@ -0,0 +1,60 @@
+const { Sequelize, DataTypes } = require('sequelize');
+const sequelize = require('../db');
+
+const Item = sequelize.define('Item' , {
+     id: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        defaultValue: Sequelize.UUIDV4
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      description: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      price: {
+        type: DataTypes.FLOAT,
+        allowNull: false
+      },
+      qty: {
+        type: DataTypes.INTEGER,
+        allowNull: false
+      },
+      imageUrl: {
+        type: DataTypes.STRING,
+        allowNull: true
+      },
+      categoryId: {
+        type: DataTypes.UUID,
+        references: {
+          model: 'Categories',
+          key: 'id'
+        },
+        onUpdate: 'CASCADE',
+        onDelete: 'SET NULL'
+      },
+      userId: {
```

Comment:
remove userId field from Item schema since it won't have a direct relationship to User schema